src/App.js

```diff
@@ -1,11 +1,15 @@
 import './App.css';
+import React from 'react';
+import Calculator from './calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
**Please let's stick to the project requirement, kindy double-check if you're following the instructions in the pathwright, and also do well to follow the naming conversion.**

We need to render the Display and ButtonPanel components in `App.js`, and also you need to wrap both `Display` and `ButtonPanel` with [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax)
Screenshot from pathwright:
![image](https://user-images.githubusercontent.com/31897434/130064376-9bdc9cba-aa76-4527-8882-a45d8c52ca86.png)


Reason for making use of React Fragment:
We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.